docs/internal/config/Template-Builder.md

```diff
@@ -1,15 +1,121 @@
+
+Primary defining the plugin type is required for a field. Other imporatant common properties are a `title`, `code`, `category`, `is_reviewable`, `help_tip` and `validation_message`. The description for each of the common fields is described in [Template element fields specs](Element-Type-Specs#template-element-fields.md).
+
+![Form field - common properties](../images/Template-builder7.png)
+
+Each plugin will have specific parameters to be passed on. These can be a simple string, number or have more elegant and complex logic, using the [`evaluateExpression`](https://github.com/openmsupply/conforma-server/wiki/Query-Syntax) function. To use these a lot of expertise is required, we definitelly  recommend having TMF support while working on this area of the App.
+
+The compulsory and optional parameters are displayed at the top:
+![Form field - parameters](../images/Template-builder8.png)
+
+## Permissions tab
+
+This area is used to define permissions used to **Apply** and **Review** applications of this type.
+
+Is simple to define permissins to Apply or View the applications. Just select one existing permission from the dropdown to use an existing **permission_name**.
+A apply permission will allow an Applicant with the given permission to create a new application. And the View is meant to be for Supervisors of reviews to View ALL submitted reviews related to applications of this template type. 
+![Permission tab - apply permission](../images/Template-builder-permissions1.png)
+
+The reviews configuration on the other hand, have many options available for procedures. In the example below we can see how it looks having just one review **stage** and a single level - meaning there is no consolidation involved. The user can start editing the current existing stage (or add a new stage) and:
+* Set the assigner permission to assign reviewers in that level/stage
+![Permission tab - assign permission](../images/Template-builder-permissions2.png)
+
+* Clicks on the plus button to add the new Level of review
+![Permission tab - level](../images/Template-builder-permissions3.png)
+
+* Select the reviewer permission to be assigned to this level/stage 
+![Permission tab - reviewer](../images/Template-builder-permissions4.png)
+
+* Then by clicking the add button next will add the new Review permission
+![Permission tab - review permission](../images/Template-builder-permissions5.png)
+## Actions tab
+
+Some of the **Core-actions** will be defined for every reviewable application, and don't show currently in this tab.
+
+What can be definied in the actions tab is anything specific to this application that should happen when the Trigger in list happen.
+Please see [Triggers & Actions](https://github.com/openmsupply/conforma-server/wiki)Triggers-and-Actions.md) for an explanation of how the trigger/action system works. Some common specific actions are: `SendNotification`, `modifyRecord` and `addUserToOrganisation` as showed in the example.
```

Comment:
Case: `sendNotification`

Also, there's an extra ")" in that link that makes it not render correctly.

Reply:
Also missing paragraph break